OptimizelySDKEventDispatcher/OptimizelySDKEventDispatcher/OPTLYEventDispatcher.m

```diff
@@ -127,27 +127,25 @@ - (OPTLYDataStore *)dataStore {
 // The timer must be dispatched on the main thread.
 - (void)setupNetworkTimer:(void(^)(void))completion
 {
+    __weak typeof(self) weakSelf = self;
     dispatch_block_t block = ^{
-        if (self.eventDispatcherDispatchInterval > 0) {
-            self.timer = [NSTimer scheduledTimerWithTimeInterval:self.eventDispatcherDispatchInterval
-                                                          target:self
-                                                        selector:@selector(flushEvents)
-                                                        userInfo:nil
-                                                         repeats:YES];
-            
-            NSString *logMessage =  [NSString stringWithFormat: OPTLYLoggerMessagesEventDispatcherNetworkTimerEnabled, self.eventDispatcherDispatchInterval];
-            [_logger logMessage:logMessage withLevel:OptimizelyLogLevelDebug];
-            
+        typeof(self) strongSelf = weakSelf;
```

Comment:
I know this is a nit but could you change it so that you assign strong self and then do a 
if (!strongSelf) return; 
instead of wrapping the if.